src/main/java/net/earthcomputer/clientcommands/mixin/CreativeInventoryScreenAccessor.java

```diff
@@ -0,0 +1,19 @@
+package net.earthcomputer.clientcommands.mixin;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+@Mixin(targets = "net.minecraft.client.gui.screen.ingame.CreativeInventoryScreen")
+public interface CreativeInventoryScreenAccessor {
+
+    @Accessor("selectedTab")
+    static void setSelectedTab(int selectedTab) {
+        throw new AssertionError();
+    }
+
+    @SuppressWarnings("AccessorTarget")
```

Comment:
You can use `@Dynamic` here instead

Reply:
Why was this resolved?

Reply:
Because I added the `@Dynamic` annotation. For your information, if I remember correctly, the `@Dynamic` annotation can't be used _instead of_ the `@SuppressWarnings` annotation: it's complementairy.